nix_dump_cst_json/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "nix_dump_cst_json"
+version = "0.1.0"
+edition = "2018"
```

Comment:
Why 2018?

Reply:
`edition` here is the Rust edition, and the latest seems to be 2018.